concourse/pipeline-v5.yml

```diff
@@ -0,0 +1,47 @@
+---
+resources:
+  - name: fauna-js-repository
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/fauna/faunadb-js.git
+      branch: v5
+
+  - name: fauna-js-repository-docs
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/fauna/faunadb-js.git
+      branch: gh-pages-v5
```

Comment:
This can be tricky, not sure we can do it that way,
for jvm driver, we have a separate folder for each version:
we could, probably, do something similar: https://github.com/fauna/faunadb-jvm/tree/gh-pages